mountpoint-s3-crt-sys/UPDATING_CRT.md

```diff
@@ -0,0 +1,39 @@
+# CRT Submodules
+
+Mountpoint for Amazon S3 imports the required [AWS Common Runtime (CRT)](https://docs.aws.amazon.com/sdkref/latest/guide/common-runtime.html) libraries through git submodules, located under the `mountpoint-s3-crt-sys/crt` directory.
+
+Currently the imported libraries are:
+
+* `aws-c-auth`
+* `aws-c-cal`
+* `aws-c-common`
+* `aws-c-compression`
+* `aws-c-http`
+* `aws-c-io`
+* `aws-c-s3`
+* `aws-c-sdkutils`
+* `aws-checksums`
+* `aws-lc`
+* `s2n-tls`
+
+## Update Submodules
+
+The CRT submodules can be updated by following these steps:
+
+1. Update every submodule to the latest tagged release. E.g. by running:
+
+   ```sh
+   git submodule foreach -q 'git checkout `git tag -l --sort=-v:refname | head -1`'
+   ```
```

Comment:
I think we need to add in a fetch here. Also, let's not suppress the `submodule foreach` log telling us which submodule the update is applied to - I think it'll help make the output clearer.

```suggestion
1. Update every submodule to the latest tagged release. E.g. by running:

   ```sh
   git submodule foreach 'git fetch -q --tags && git checkout `git tag -l --sort=-v:refname | head -1`'
   ```
```

Reply:
You're right. I only meant to add it to the "check" command where we already print the submodule path.